src/current/v25.1/time.md

```diff
@@ -180,7 +180,7 @@ ALTER TABLE
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> SHOW COLUMNS FROM time;
+> SHOW COLUMNS FROM time_precise;
 ~~~
 
 ~~~
```

Comment:
The output I got:

~~~
  column_name | data_type | is_nullable | column_default | generation_expression |       indices       | is_hidden
--------------+-----------+-------------+----------------+-----------------------+---------------------+------------
  time_id     | INT8      |      f      | NULL           |                       | {time_precise_pkey} |     f
  time_val    | TIME(5)   |      t      | NULL           |                       | {time_precise_pkey} |     f
~~~

Reply:
updated! thanks